pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -188,11 +205,25 @@ async def run(self):
                 self._jira_client.close_task(subtask)
                 self._jira_client.start_task(jira_issue)
             else:
-                _LOGGER.warning("[DRY RUN ]Would have updated Jira ticket status")
+                _LOGGER.warning("[DRY RUN] Would have updated Jira ticket status")
 
         _LOGGER.info("Updating ocp-build-data...")
         await self.update_build_data(advisories, jira_issue_key)
 
+        if "advance" in advisories.keys():
+            advisory_info = await self.get_advisory_info(advisories["advance"])
+            # Make sure that the advisory is in editable mode
+            if self.is_advisory_editable(advisory_info):
+                # Set this as an 'advance' release
+                self.advance_release = True
+
+                # Remove all builds and bugs from the metadata advisory
+                await self.remove_builds_all(advisories["metadata"])
+                await self.remove_bugs_all(advisories["metadata"])
```

Comment:
let's remove this since the bugs workflow is different from what we thought it was
```suggestion
              
```